src/main/java/org/fcrepo/importexport/common/Config.java

```diff
@@ -134,6 +135,22 @@ public boolean isIncludeBinaries() {
     }
 
     /**
+     * Sets the location to write a sample config file.
```

Comment:
We should probably include this arg in the config file, itself, no?
https://github.com/fcrepo4-labs/fcrepo-import-export/blob/master/src/main/java/org/fcrepo/importexport/common/Config.java#L465

...as well as the tests:
- https://github.com/fcrepo4-labs/fcrepo-import-export/blob/master/src/test/java/org/fcrepo/importexport/ArgParserTest.java#L329
- https://github.com/fcrepo4-labs/fcrepo-import-export/blob/master/src/test/java/org/fcrepo/importexport/ArgParserTest.java#L357